recipe/activate-msvc-headers-libs.sh

```diff
@@ -18,33 +18,19 @@ if [[ ! -d "${MSVC_HEADERS_DIR}" ]]; then
   pushd "${MSVC_HEADERS_DIR}"
     mkdir -p tmp
     pushd tmp
-      # These URLs come out os the HTTP API used by vs_buildtools.exe.
-      # You can use https://github.com/mstorsjo/msvc-wine/blob/9ebc76c16e5849438ddb6566042af87dcff0ca1e/vsdownload.py
-      # to download these files programatically. We are only interested in the subset that creates the headers and x64
-      # libraries in Contents/VC/Tools/MSVC/${MSVC_HEADERS_VERSION}/
-      #
-      # To find the download URLs, do
-      #  1. Run a full download using python vsdownload.py --msvc-15.9 --accept-license  --dest . --major 16 Microsoft.VisualStudio.Component.VC.v141.x86.x64
-      #  2. Use `(rip)grep 'Contents/VC/Tools/MSVC/${MSVC_HEADERS_VERSION}/'` to determine all *.vsix's that have contain relevant content.
-      #  3. Add debug output to https://github.com/mstorsjo/msvc-wine/blob/9ebc76c16e5849438ddb6566042af87dcff0ca1e/vsdownload.py#L415
-      #     to fine the correct `payload["url"]` matching the vsix filenames in `fileid`.
-      #  4. Update the URLs in this script.
+      # these get determined & filled in by install-pkg.sh,
+      # which produces a list of "component|url" pairs
+      declare -a COMPONENT_URLS=(
+        @COMPONENT_URLS@
+      )
+
+      for map in "${COMPONENT_URLS[@]}"; do
+        # component=$(echo $map | cut -d '|' -f1)
+        url=$(echo $map | cut -d '|' -f2)
+        # download and unpacks into Contents/VC/Tools/MSVC/${MSVC_HEADERS_VERSION}/...
+        curl -L $url -o tmp.zip && unzip -oqq tmp.zip && rm tmp.zip
```

Comment:
After I had scripted all the automation, I ended up not using the component name explicitly in the activation script. I still think it's a value add to keep it, as that makes the activation script way more self-explanatory _with_ the component names in `COMPONENT_URLS`, than if it were just a list of obfuscated URLs.